test/deleteUsers/data/mp/handler_input.json

```diff
@@ -3083,5 +3083,53 @@
         }
       ]
     }
+  },
+  {
+    "request": {
+      "body": [
+        {
+          "destType": "MP",
+          "userAttributes": [
+            {
+              "userId": "rudder1"
```

Comment:
Have we considered the case where we will get other user attributes inside the object?

Reply:
we are not using other attributes.

Reply:
What if another destination's deletion function is using other attributes and the same event came in Mixpanel?

Reply:
Then we will only take `userId` for calculation.